common/credential-storage/src/backends/sqlite.rs

```diff
@@ -28,50 +28,45 @@ impl CoconutCredentialManager {
     ) -> Result<(), sqlx::Error> {
         sqlx::query!(
             r#"
-                INSERT INTO coconut_credentials(serialization_revision, credential_type, credential_data, epoch_id, expired)
-                VALUES (?, ?, ?, ?, false)
+                INSERT INTO ecash_credentials(serialization_revision, credential_type, credential_data, epoch_id, expired, consumed)
+                VALUES (?, ?, ?, ?, false, false)
             "#,
             serialization_revision, credential_type, credential_data, epoch_id
         ).execute(&self.connection_pool).await?;
         Ok(())
     }
 
-    pub async fn get_next_unspect_freepass(
+    pub async fn get_next_unspent_freepass(
```

Comment:
but how to we guarantee this freepass hasn't been spent with the desired gateway?

Given how many times I've mentioned it, I feel we will talk about it : P